packages/prerender-proxy/lib/prerender-lambda-construct.ts

```diff
@@ -23,5 +26,6 @@ export class PrerenderLambda extends Construct {
     
     this.errorResponseFunction = new ErrorResponseFunction(this, 'ErrorResponse', {});
 
+    this.cacheControlFunction = new CloudFrontCacheControl(this, 'PrerenderCloudFrontCacheControl', props.cacheControlProps)
```

Comment:
Should we have an option to disable this function?

Reply:
Good question, should we?  As I understand this is a flaw?

Reply:
I'm just thinking for existing builds. Always nice to have the option to fall back to what we previously had. Although going forward I feel like most builds will implement this. 
I just like options :)

Reply:
Hmm, for the existing implementations, it will only create the function but you will have to explicitly introduce this when you use this construct. 
Should this be a problem?

Reply:
For future reference, we had a discussion about this and as only creates the lambda function and doesn't automatically add it to the behaviour it's fine to not have an option to disable.